index.html

```diff
@@ -9,11 +9,9 @@
 <body>
   <h1>Javascript Quiz</h1>
   <form action="#" onsubmit="tree()">
```

Comment:
No forms, and no running javascript from the html. We'll get into that later. The reason for no form is because when you press `Enter` it automatically submits. We want you to hand code the `Enter` press. Then you want to set up one or a few event listeners to handle the submission.